horovod/common/operations.cc

```diff
@@ -133,44 +141,75 @@ const Status DUPLICATE_NAME_ERROR = Status::InvalidArgument(
     "to request another tensor, use a different tensor name.");
 
 OperationManager* CreateOperationManager(HorovodGlobalState& state) {
-  // Order of these operations is very important. Operations will be checked sequentially from the first
-  // to the last. The first 'Enabled' operation will be executed.
+  // Order of these operations is very important. Operations will be checked
+  // sequentially from the first to the last. The first 'Enabled' operation will
+  // be executed.
   std::vector<std::shared_ptr<AllreduceOp>> allreduce_ops;
   std::vector<std::shared_ptr<AllgatherOp>> allgather_ops;
   std::vector<std::shared_ptr<BroadcastOp>> broadcast_ops;
 
 #if HAVE_CUDA
 #if HOROVOD_GPU_ALLREDUCE == 'M'
-  allreduce_ops.push_back(std::shared_ptr<AllreduceOp>(new MPI_CUDAAllreduce(&mpi_context, &cuda_context, &state)));
+  allreduce_ops.push_back(std::shared_ptr<AllreduceOp>(
+      new MPI_CUDAAllreduce(&mpi_context, &cuda_context, &state)));
 
 #else
-  #if HAVE_NCCL && HOROVOD_GPU_ALLREDUCE == 'N'
-    allreduce_ops.push_back(std::shared_ptr<AllreduceOp>(
-        new NCCLHierarchicalAllreduce(&nccl_context, &mpi_context, &cuda_context, &state)));
-    allreduce_ops.push_back(std::shared_ptr<AllreduceOp>(
-        new NCCLAllreduce(&nccl_context, &mpi_context, &cuda_context, &state)));
-
-  #elif HAVE_DDL && HOROVOD_GPU_ALLREDUCE == 'D'
-    allreduce_ops.push_back(std::shared_ptr<AllreduceOp>(new DDLAllreduce(&ddl_context, &cuda_context, &state)));
-  #endif
+#if HAVE_NCCL && HOROVOD_GPU_ALLREDUCE == 'N'
+  LOG(INFO)<<"NCCL enabled.\n";
```

Comment:
No need to put newline characters at the end, the logging framework will handle that for you.